### ISSUE TITLE
Consider unifying this repo with upstream

### ISSUE THREAD

**Issue by mattip** [MacPython/statsmodels-wheels#162]
Since the migration to cibuildwheel, it would seem this repo could be merged with the upstream statsmodels. That would get more eyes on the wheel building, and would make it easier to manage things like the API secret by the project maintainers. Is there a reason to keep it separate here?

**Comment by bashtage** [comment]
We should probably just migrate it to the statsmodels org as a speerate repo.  I agree there isn't much of a case to continue to use this org.

**Comment by agriyakhetarpal** [comment]
I fully agree with this, it took me a short while to find out where statsmodels was publishing its nightly wheels from! This repository is hidden in the nooks and crannies of a separate organisation :)

**Comment by agriyakhetarpal** [comment]
I fully agree with this, it took me a short while yesterday to find out where statsmodels was publishing its nightly wheels from! This repository is hidden in the nooks and crannies of a separate organisation :)

**Comment by josef-pkt** [comment]
numpy, scipy and pandas have already moved.

So, I don't see a reason to move statsmodels builds also.

@bashtage 
How should we do this?
for example 
https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-organization
or just forking, or creating a separate repo 

AFAIU, you have permission on both organizations to make a transfer.

**Comment by agriyakhetarpal** [comment]
It could also be moved and maintained as a standalone workflow in the main statsmodels repository, which would avoid the need for updating the submodule before building.

**Comment by mattip** [comment]
Ping. Also is the project still using https://anaconda.org/multibuild-wheels-staging/statsmodels ? Most other projects have stopped uploading there, and are using trusted publishing (edit: or downloading the wheels locally) instead.

**Comment by bashtage** [comment]
Yes.  We used it on Friday for 0.14.6.  what are others doing with releases?
